examples/counter/src/service.rs

```diff
@@ -25,11 +26,44 @@ impl Service for Counter {
     ) -> Result<Vec<u8>, Self::Error> {
         match argument {
             &[] => Ok(bcs::to_bytes(&self.value).expect("Serialization should not fail")),
-            _ => Err(Error::InvalidQuery),
+            bytes => {
+                let graphql_request: async_graphql::Request =
+                    serde_json::from_slice(bytes).map_err(|_| Error::InvalidQuery)?;
+                let schema = Schema::build(
+                    QueryRoot { value: self.value },
+                    MutationRoot {},
+                    EmptySubscription,
+                )
+                .finish();
+                let res = schema.execute(graphql_request).await;
+                Ok(serde_json::to_vec(&res).unwrap())
+            }
         }
     }
 }
 
+struct MutationRoot;
+
+#[Object]
+impl MutationRoot {
+    #[allow(unused)]
+    async fn execute_operation(&self, operation: u64) -> Vec<u8> {
```

Comment:
Should we call this `increment`?

Reply:
Yes, I'll do this in a new PR since this one is already quite big. #696